docs/modules/migrate/partials/nav.adoc

```diff
@@ -1,5 +1,6 @@
 ** xref:deploy:versioning-compatibility.adoc[Versioning and Compatibility]
 ** xref:migrate:data-migration-tool.adoc[]
+** xref:migrate:dmt-command-reference.adoc[]
```

Comment:
***

(to make it a section under the DMT section, not a new section at the same level)

Reply:
https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1